packages/react/src/components/SidePanel/SidePanel.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Close16,
+  ChevronLeft16,
+  ChevronRight16,
+  OpenPanelLeft16,
+  OpenPanelRight16,
+} from '@carbon/icons-react';
+import classNames from 'classnames';
+
+import Button from '../Button';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+const propTypes = {
+  direction: PropTypes.string,
+  open: PropTypes.bool,
+  slideOver: PropTypes.bool,
+  inline: PropTypes.bool,
+  slideIn: PropTypes.bool,
+  showCloseButton: PropTypes.bool,
+  showDrawer: PropTypes.bool,
```

Comment:
we may want to align with the `SideNav` that provides [similar control](https://github.com/carbon-design-system/carbon/blob/a48af3011622ae163e70324e140da9510cb636f6/packages/react/src/components/UIShell/SideNav.js#L214) and rename this prop `isRail`

Reply:
Because at first this sounds like what `open` prop is.

Reply:
FYI, [Here](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/60e9f1ea74e52e824d59856e0368d94ae53b1090/packages/react/docs/guides/onboarding.md#prop-naming-conventions) are our docs on prop naming conventions. We really need to make these docs more discoverable.